### ISSUE TITLE
Unable to use IPv6

### ISSUE THREAD

**Issue by skmike** [kimchi-project/kimchi#654]
I've pursuaded nginx to listen on IPv6 by changing the 2 listen lines:
listen [::]:${proxy_ssl_port} ssl;
listen [::]:${proxy_port};

However I don't know how to persuade the python daemon to listen to IPv6...

**Comment by ramonmedeiros** [comment]
Hi,
 
have you tried to change /etc/kimchi/kimchi.conf? :

[server]
# Hostname or IP address to listen on
#host = 0.0.0.0

# Port to listen on
#port = 8000

# If present, start an SSL-enabled server on the given port
#ssl_port = 8001

# Cherrypy server port
#cherrypy_port = 8010

**Comment by skmike** [comment]
Yes - I've changed the host to both :: and a full IPv6 address with no joy.

For the nginx config that should have read "added" instead of "changing" as both pairs of lines are required.
